web/src/engine/websites/Mangadon.ts

```diff
@@ -67,7 +78,10 @@ export default class extends DecoratableMangaScraper {
     }
 
     public override async FetchPages(chapter: Chapter): Promise<Page[]> {
+        await this.checkAuth();
+
         const request = new Request(new URL(`/api/v1/episodes/${chapter.Identifier}?params%5Bcookie_signer%5D=true&params%5Bpages%5D=true`, this.URI));
```

Comment:
Does the `cookie_signer` parameter affect the auth mechanism?

Reply:
No. Its the other way around.

1) You must always get cookies from the api, logged or not, free chapter or not. They are needed for the fetch request on pictures. Only for that purpose.

2) If you have no auth, you can get cookies for pictures ONLY on FREE chapters (free meaning no logging). In case of not authorized and login needed, cookies_signer object in response is empty.

3) Its the kind of api where you specify what things you wants returned.  To fetch pictures later, "cookies_signer" param is mandatory, so we get Cloudfront cookies